wa/workloads/speedometer/__init__.py

```diff
@@ -204,6 +204,7 @@ def setup(self, context):
                 '<boolean name="first_run_tos_accepted" value="true" />\n',
                 '<boolean name="first_run_signin_complete" value="true" />\n',
                 '<boolean name="displayed_data_reduction_promo" value="true" />\n',
+                '<int name="Chrome.NotificationPermission.RequestCount" value="2" />\n',
```

Comment:
nit: Would it be worth adding a comment to the code so we know what this line had been added to achieve going forward?